functests/performance/rt-kernel.go

```diff
@@ -59,4 +60,19 @@ var _ = Describe("[performance]RT Kernel", func() {
 
 	})
 
+	It("[test_id:28526][crit:high][vendor:cnf-qe@redhat.com][level:acceptance] Non worker-cnf node should not have RT kernel installed", func() {
+
+		By("Skipping test if cluster does not have another available worker node")
+		nonRTWorkerNode, err := nodes.GetNonRTWorkers()
+		Expect(err).ToNot(HaveOccurred())
+
+		if len(nonRTWorkerNode) == 0 {
+			Skip("Skipping test because there are no additional non-cnf worker nodes")
+		}
+
+		cmd := []string{"uname", "-a"}
+		Expect(execCommandOnWorker(cmd, &nonRTWorkerNode[0])).To(ContainSubstring("Linux"), "Node should have Linux string")
```

Comment:
you can run `execCommandOnWorker` once only and use the result in both `Expect`s